### PR TITLE
Bump version number to v0.13

### PR DESCRIPTION
**Pull request by josh146**

### CODE REVIEW

File: qchem/setup.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     version = f.readlines()[-1].split()[-1].strip("\"'")
 
 requirements = [
-    "pennylane>=0.12",
+    "pennylane>=0.13",
```
Comment (antalszava) [inline qchem/setup.py:20]: Think this will have to be pinned in `requirementst.txt` too.

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Release 0.13.0-dev (development release)
+# Release 0.13.0-dev (current release)
```
Comment (antalszava) [inline .github/CHANGELOG.md:1]:
```suggestion
# Release 0.13.0 (current release)
```

**Review by co9olguy — COMMENTED** [review]
Thanks @antalszava for the careful review!

**Review by antalszava — COMMENTED** [review]
Hi @josh146, looking nice! :slightly_smiling_face: 

A couple of thoughts:
* Think the `__version__ = "0.13.0-dev"` -> `__version__ = "0.13.0"` change has to be made in `pennylane_qchem/_version.py`.
* Just realized that there is a `CHANGELOG` file for `qchem`, there might be an entry to the bug fix section (can push a commit if you'd like me to):
```markdown
<h3>Bug fixes</h3>

* The `qchem._terms_to_qubit_operator` function is now updated to handle tensor products with `Identity` observables.
  [(#928)](https://github.com/PennyLaneAI/pennylane/pull/928)
```
* Just a heads-up: the formulas included for the change in parameter-shift gradient recipes on lines 360-366 will not render, but the latex will be shown (perhaps intended :+1:).

**Review by antalszava — APPROVED** [review]
Looks good to me! :rocket: :100:

**Pull request merged**